### ISSUE TITLE
[Bug Report] Download Manager Broken (https?)

### ISSUE THREAD

**Issue by DJScias** [hkalexling/Mango#144]
**Describe the bug**
Well, the Download Manager page is broken.
![image](https://user-images.githubusercontent.com/439655/104210643-6933cd80-5433-11eb-8582-1e267dcd16da.png)

The console says this: 
![image](https://user-images.githubusercontent.com/439655/104210687-73ee6280-5433-11eb-8c5c-5c17a765f193.png)
and
```
Uncaught DOMException: Failed to construct 'WebSocket': An insecure WebSocket connection may not be initiated from a page loaded over HTTPS.
```
I use a reverse-proxy (you've logged into it before) that uses https. Is this related to it perhaps?

**To Reproduce**
1. Run Mango as a reverse-proxy (in Synology) with https (letsencrypt)
2. Go to Download Manager
2. Watch it be broken

**Expected behavior**
It just working.

**Environment (please complete the following information):**
- OS: Synology DS918+, DSM 6.2.3-25426 Update 3
- Browser: Brave (chromium-based)
- Mango Version: v0.18.2

**Docker (if you are running Mango in a Docker container)**
```
sudo docker run \
  -d \
  --name=mango \
  -e PUID=1024 \
  -e PGID=100 \
  -p 9000:9000/tcp \
  -v /volume1/docker/Mango/config:/root/.config/mango \
  -v /volume4/Plex4/Mango:/root/mango \
  --restart always \
	hkalexling/mango:v0.18.2
```

**Additional context**
Screenshots added to bug description.

**Comment by hkalexling** [comment]
Oh, you are right 🤦  Only the secure WebSocket (`wss://`) is allowed under HTTPS. I will push a bug fix release soon.

**Comment by hkalexling** [comment]
This is now fixed in [v0.18.3](https://github.com/hkalexling/Mango/releases/tag/v0.18.3). Thanks a lot for the bug report!

**Comment by DJScias** [comment]
Perfect, it all works now!

Thank you for the swift fix.
